src/providers/postgres/qgspostgresprovider.cpp

```diff
@@ -3890,144 +3885,189 @@ QgsRectangle QgsPostgresProvider::extent() const
   return extent3D().toRectangle();
 }
 
-QgsBox3D QgsPostgresProvider::extent3D() const
+bool QgsPostgresProvider::estimateExtent() const
 {
-  if ( !isValid() || mGeometryColumn.isNull() )
-    return QgsBox3D();
+  // Cannot estimate extent of a query
+  if ( mIsQuery )
+  {
+    QgsDebugMsgLevel( tr( "Estimating extent of queries is not supported" ), 2 );
```

Comment:
```suggestion
    QgsDebugMsgLevel( "Estimating extent of queries is not supported", 2 );
```

Debug messages are not translated

Reply:
Pushed a commit dropping the translations, thanks for the review!